Documentation/HowToGuides/CreatingADoor/README.md

```diff
@@ -0,0 +1,135 @@
+* [Install the Tilia.Interactions.Controllables.Unity] package dependency in to your Unity project.
+
+## Let's Start
+
+### Step 1
+
+Create a new `Empty` GameObject by selecting `Main Menu -> GameObject -> Create Empty` and change the Transform properties to:
+
+* Position: `X = 0, Y = 0, Z = 0`
+* Scale: `X = 1, Y = 1, Z = 1`
+
+Rename the new empty `GameObject` to `DoorFrame`.
+
+![Create DoorFrame](assets/images/CreateDoorFrame.png)
+
+### Step 2
+
+Right click on the `DoorFrame` GameObjet, select `3D Object -> Cube` and change the Transform properties to:
+
+* Position: `X = 0, Y = 1, Z = 0`
+* Scale: `X = 1.1, Y = 0.1, Z = 0.1`
+
+Rename the new empty `Cube` to `TopFrame`.
+
+![Create TopFrame](assets/images/CreateTopFrame.png)
+
+> It may be easier to apply a different material to the cabinet to make it easier to see.
+
+### Step 3
+
+Create the remaining components of the `DoorFrame` by duplicating the `TopFrame` GameObject 2 times and for each duplicated GameObject change the Transform properties to:
+
+#### TopFrame (1)
+
+* Position: `X = -0.5, Y = 0, Z = 0`
+* Scale: X = `0.1, Y = 2, Z = 0.1`
+
+Rename the duplicated `TopFrame (1)` to `LeftFrame`.
+
+#### TopFrame (2)
+
+* Position: `X = 0.5, Y = 0, Z = 0`
+* Scale: `X = 0.1, Y = 2, Z = 0.1`
+
+Rename the duplicated `TopFrame (2)` to `RightFrame`.
+
+![Frame Built](assets/images/FrameBuilt.png)
+
+> It may be easier to apply a different material to the cabinet to make it easier to see.
+
+### Step 4
+
+Expand the `Tilia Interactions Controllables Unity` Package directory in the Unity Project window and select the `Packages -> Tilia Interactions Controllables Unity -> Runtime -> Prefabs -> PhysicsJoint` directory then drag and drop the `Interactions.AngularJointDrive` prefab into the Unity hierarchy window so it becomes a child of the `DoorFrame` GameObject.
+
+![Import Angular Joint Drive](assets/images/ImportAngularJointDrive.png)
+
+> The `Interactions.AngularJointDrive` prefab uses Unity joints and therefore works within the Unity physics system, however the `Interactions.AngularTransformDrive` is an angular drive that does not utilize joints or physics and can easily be swapped in place at this step if required.
+
+### Step 5
+
+Select the `DoorFrame -> Interactions.AngularJointDrive -> Internal -> JointContainer -> Joint -> Interactions.Interactable -> MeshContainer -> Cube` GameObject from the Unity Hierarchy and change the Transform properties to:
+
+* Position: `X = 0, Y = -0.025, Z = 0`
+* Scale: `X = 0.9, Y = 1.95, Z = 0.05
+
+![Set Up Door Mesh](assets/images/SetUpDoorMesh.png)
+
+### Step 6
+
+Select the `DoorFrame -> Interactions.AngularJointDrive` GameObject from the Unity Hierarchy and on the `Angular Drive Facade` component set the following properties to:
+
+* Drive Axis: `Y Axis`
+* Drive Limit: `Min: -120, Max: 120`
+* Hinge Location: `X = -0.45`
+
+![Set Up Angular Joint Drive Facade](assets/images/SetUpAngularJointDriveFacade.png)
+
+### Step 7
+
```

Comment:
Stick some blurb in as to why we need to use the collision ignorer

Reply:
blurb to add:

```
We're now going to have an issue with the door control set up in the door frame because both the door and door frame have colliders. This means the door will collide with the door frame and start having adverse effects. We could be really particular with our door and door frame design so the colliders do not intersect or we could cheat a little bit and just get the door frame and door to ignore collisions between each other.

We can do this using the `Mutators.CollisionIgnorer` prefab which will tell all of the colliders in the door frame to ignore all of the colliders in the door.
```